extensions/blocks/opentable/opentable.php

```diff
@@ -72,6 +73,19 @@ function load_assets( $attributes ) {
 		$classes
 	);
 	$content = '<div class="' . esc_attr( $classes ) . '">';
+
+	// OpenTable's wide style has fixed widths within the embedded iframe.
+	// To improve the mobile experience we'll add a standard style widget as
+	// well and use CSS to switch display between the wide style and it.
+	if ( 'wide' === $style ) {
```

Comment:
If it's only on mobile, maybe we could rely on an additional `jetpack_is_mobile` check as well?